Devscord.DiscordFramework/Middlewares/Factories/MessageContextFactory.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Discord;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using System;
+
+namespace Devscord.DiscordFramework.Middlewares.Factories
+{
+    public class MessageContextFactory : IContextFactory<IUserMessage, MessageContext>
+    {
+        private readonly UserContextsFactory _userContextsFactory;
+
+        public MessageContextFactory()
+        {
+            this._userContextsFactory = new UserContextsFactory();
+        }
+
+        public MessageContext Create(IUserMessage userMessage)
+        {
+            if (userMessage == null)
+            {
+                throw new ArgumentNullException();
```

Comment:
a co sie dzieje przy wyrzucaniu tutaj tego wyjątku?
jak to później obsługujemy?